moderation/assets/moderation.html

```diff
@@ -472,6 +472,8 @@ <h4>Delete ALL server warnings?</h4>
 
         {{checkbox "WarnIncludeChannelLogs" "WarnIncludeChannelLogs" "Create message logs in the channel that the command was run in when a user is warned" .ModConfig.WarnIncludeChannelLogs}}
         {{checkbox "WarnSendToModlog" "WarnSendToModlog" "Send warnings to the modlog" .ModConfig.WarnSendToModlog}}
+        {{checkbox "UnwarnSendToModlog" "UnwarnSendToModlog" "Send warning removal to the modlog" .ModConfig.UnwarnSendToModlog}}
+        {{checkbox "UnwarnIncludeWarnReason" "UnwarnIncludeWarnReason" "Include warning reason in the modlog" .ModConfig.UnwarnIncludeWarnReason}}
```

Comment:
I’m not sure it’s immediately evident this is still referring to a warn removal.

Reply:
Mhh, the command being called deletewarning, it doesn't make sense at all

I'll change that

Reply:
Any idea on the text to show ?

Reply:
![image](https://github.com/user-attachments/assets/781ad177-277a-4f3a-860a-e8c85d6885c3)
for now

Reply:
And in back, all renamed to Delwarnxxxxx to match the command name

Reply:
“Append original warn reason to removal” :maby:

Reply:
I don't like replacing "removing" by "deletion" but it matches the function....

Reply:
yep that makes sense to me